downstream/modules/hub/proc-create-api-token-pah.adoc

```diff
@@ -17,14 +22,11 @@ In {PrivateHubName}, you can create an API token using API token management. The
 . To copy the API token, click the btn:[Copy to clipboard] icon.
 . Paste the API token into a file and store in a secure location.
 
-[IMPORTANT]
+[NOTE]
 ====
-The API token is a secret token used to protect your content. Store your API token in a secure location.
+The API token does not expire.
 ====
 
+.Next step
```

Comment:
Just a nit, but might be worth noting. This seems more like `.Results` rather than `.Next step` as it is not an imperative statement requiring the customer to take an action. Of course, it's your final decision.